tools/rpi-eeprom-digest

```diff
@@ -0,0 +1,108 @@
+#!/bin/sh
+
+# Helper script to generate .sig files for use with the Raspberry Pi bootloader.
```

Comment:
You may wish to add a `4` after `Raspberry Pi` to make this consistent with the rest of the documentation, usage messages in other utilities etc.